DesignRules.md

```diff
@@ -540,12 +540,11 @@ Changes in APIs are inevitable. APIs should therefore always be versioned, facil
          <p>An example of a base path for an API with current version 1.0.2:</p>
          <pre>https://api.example.org/v1/</pre>
          <pre>version: '1.0.2'</pre>
-         <pre>servers:
-                  - description: test environment  
-                  url: https://api.test.example.org/v1/  
-                  - description: production environment  
-                  url: https://api.example.org/v1/  
-         </pre>
+         <pre><code class="yaml">servers:
+   - description: test environment  
+   url: https://api.test.example.org/v1/  
+   - description: production environment  
+   url: https://api.example.org/v1/</code></pre>
```

Comment:
Deze ziet er nu goed uit, maar is de `<code>` block nodig, of zou `class="yaml"` ook direct in `<pre>` kunnen?

Reply:
https://highlightjs.org/ zegt

> This will find and highlight code inside <pre><code> tags;

Dus ik denk dat allebei inderdaad nodig zijn. Misschien dat het anders ook werkt, maar ik houd het liever op de officiele manier